Templates/AppSource App/.AL-Go/cloudDevEnv.ps1

```diff
@@ -59,7 +59,8 @@ if (-not $environmentName) {
     $environmentName = Enter-Value `
         -title "Environment name" `
         -question "Please enter the name of the environment to create" `
-        -default "$($env:USERNAME)-sandbox"
+        -default "$($env:USERNAME)-sandbox" `
+        -trimQuotesAndSpaces
```

Comment:
Why don't you simply add `$environmentName.trim(@('"',"'",' '))` after `Enter-Value`?

Adding complexity to a commodity function doesn't seem justified here, in my opinion.

If you insist on adding a new parameter, why about `$trimCharacters = @()` to cover potential future cases where we might want to trim parathesis `(` `)` or some other characters?